rust/src/load.rs

```diff
@@ -143,3 +168,19 @@ pub async fn load_vcf(vcf_path: PathBuf, db_url: &str) -> PyResult<()> {
     info!("Time taken: {:?}", duration);
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use tempfile::NamedTempFile;
+
+    #[tokio::test]
+    async fn test_load_file_uri() {
+        let temp_file = NamedTempFile::new().expect("Failed to create temp file");
```

Comment:
help me out here, so what is this file testing? I'm unfamiliar with Rust so not sure how the db file is being created

Reply:
gotcha and for my understanding, how is the sqlite:/// interpreted? I was expecting to see an file exntension like .sqlite on the filename for example.

Reply:
It's just there as a scheme declaration to indicate a sqlite file, I guess. I think it's an expectation imposed by the DB client library. Just checked and this test fails if you change it.

I don't know if there's really an accepted file extension for sqlite files (i just googled it and [this stackoverflow answer](https://stackoverflow.com/questions/808499/does-it-matter-what-extension-is-used-for-sqlite-database-files) is very unsatisfying). It's definitely not something we try to validate anywhere, although we could if it seems worthwhile.